docs/learn/codemonkey/codemonkey-complete-tutorial.md

```diff
@@ -30,7 +30,7 @@ This video  covers
 - Unity Transport
 - Creating the Player
 - NetworkObject
-- Assigning the PlayerPrefab and NetworkPrefab List
+- Assigning the PlayerPrefab and NetworkPrefab List (From NGO release 1.3.1+, the NetworkPrefabs are assigned to a ScriptableObject)
```

Comment:
I'm not sure about this. I don't really know the context. But since this says "codemonkey" I think this addition is unnecessary, since the changes only affect the editor workflow and not code.